app/views/users/profile.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1 class="mb-4 text-light"> <%= @user.name %> </h1>
```

Comment:
- [ ] Please rename this page so it follow the Rails naming conventions, this view should be called "show" and the action at the user controller #show, a show page is one that shows a single instance of a model while the index one shows a list, so since the profile page shows only a single user it should be a show page.
  - In the UI of the app itself it's ok to name this page "Profile", but to keep the code more organized let's follow the RoR conventions :-)